columnflow/production/normalization.py

```diff
@@ -331,6 +340,13 @@ def normalization_weights_init(self: Producer) -> None:
     Initializes the normalization weights producer by setting up the normalization weight column.
     """
     self.produces.add(self.weight_name)
+    if self.allow_stitching:
+        self.stitching_datasets = self.get_stitching_datasets()
+    else:
+        self.stitching_datasets = [self.dataset_inst]
+
+    if self.allow_stitching and self.dataset_inst == self.get_inclusive_dataset():
```

Comment:
From above: you can store a reference to the inclusive dataset here and use it above (also in line 299 within `setup`).